app/models/group_user.rb

```diff
@@ -8,7 +8,7 @@ class GroupUser < ActiveRecord::Base
   after_destroy :grant_other_available_title
 
   after_save :set_primary_group
-  after_destroy :remove_primary_group, :recalculate_trust_level
+  after_destroy :remove_primary_group, :remove_flair_group, :recalculate_trust_level
```

Comment:
Should we be "merging" all the `after_destroy`s? And by merging I mostly mean "group together". 

We already have a `after_destroy` on line 8

Reply:
Does that work also when bulk deleting users from a group? Is it performant?

Reply:
I wanted to keep `after_destroy` separate because they did different things, but I believe `remove_primary_group` and `remove_flair_group` can be merged because they are trivial and most of the logic is duplicated.

The callbacks are already grouped together. The group on line 7-8 is about titles, the group on 10-11 is about the primary group.

The bulk "remove member" operation simply iterates over each user and calls the `group.remove` method. It could be more performant, but these changes should not bring any slowdowns.